src/styles.scss

```diff
@@ -1341,3 +1341,12 @@ hr {
   margin: 0.5rem 0;
   border: 1px solid #dfe0e2;
 }
+
+.orgLogo {
+  height: 50px;
+  object-fit: scale-down;
+}
+
+.hidden-overflow {
```

Comment:
try:

```
@each $value in $overflows {
  .overflow-#{$value} { overflow: $value !important; }
}
```
and using class overflow-hidden and see what happens. if it works, then ur .content style can also be replaced

assuming everything works, put that in  the boostrap4 scss file

Reply:
It does, thank you!